Loop/Extensions/DosingDecisionStore+SimulatedCoreData.swift

```diff
@@ -124,6 +124,18 @@ fileprivate extension StoredDosingDecision {
                                                   pumpBatteryChargeRemaining: 3.5,
                                                   basalDeliveryState: .initiatingTempBasal,
                                                   bolusState: .none)
+        let notificationSettings = StoredDosingDecision.NotificationSettings(authorizationStatus: .authorized,
```

Comment:
For export, I created a new, Codable `StoredDosingDecision.NotificationSettings` that is stored in Core Data (rather than a binary encoded `UNNotificationSettings`). This simply reflects that change to `StoredDosingDecision`.

Reply:
Will pull out into top-level class.

Reply:
Pulled out into a top-level class in a separate file and added separate unit tests. ✅